enacts/monthly/maproom.py

```diff
@@ -91,7 +82,7 @@ def get_shapes(query):
         Input("location", "pathname"),
     )
     def initialize(path):
-        rr_mrg = read_data("precip")
+        rr_mrg = calc.read_enacts_zarr_data(**READ_PARAMS)
```

Comment:
I think I would prefer to make `variable` mandatory, rather than defaulting to `precip`. This call might be misunderstood to be fetching all the data (all variables). What's your justification for defaulting to `precip`?

Reply:
So... probably just my ignorance. I wanted arguments to be keyword arguments so that I can define them as a dictionary so that I can pass them on as parameters through the **{} calling method. If keyword, I have to give it a default value, don't I? And in that case, precip is used everywhere and was thus the more efficient/logical default.

Reply:
Positional arguments can be provided as keyword arguments.
```
>>> def foo(a, b):
...     return b - a
... 
>>> foo(b=2, a=1)
1
```